pyccel/stdlib/ndarrays/ndarrays.c

```diff
@@ -343,6 +343,22 @@ int64_t     get_index(t_ndarray arr, ...)
     return (index);
 }
 
+/*
+** size
+*/
+
+int64_t     get_size(t_ndarray arr)

```

Comment:
don't we have the size of the array as the attribute `length` in the `nd_array` struct we calculate the size of the flattened array there, we can get the size of the buffer (lenght * data_type_size) too using `buffer_size`

Reply:
I thought we had this somewhere, but I obviously didn't look very hard. Fixed : [ef6aea1](https://github.com/pyccel/pyccel/pull/858/commits/ef6aea1502a103b58773a11d6ffea0f84dc715e3)